03-transactions.md

```diff
@@ -39,6 +39,7 @@ This details the exact format of on-chain transactions, which both sides need to
     * [Storage Tests](#storage-tests)
   * [Appendix E: Key Derivation Test Vectors](#appendix-e-key-derivation-test-vectors)
   * [Appendix F: Commitment and HTLC Transaction Test Vectors (anchors)](#appendix-f-commitment-and-htlc-transaction-test-vectors-anchors)
+  * [Appendix G: Commitment and HTLC Transaction Test Vectors (anchors-zero-fee-htlc-tx)](#appendix-f-commitment-and-htlc-transaction-test-vectors-anchors-zero-fee-htlc-tx)
```

Comment:
nit: link doesn't work:
```suggestion
  * [Appendix G: Commitment and HTLC Transaction Test Vectors (anchors-zero-fee-htlc-tx)](#appendix-g-commitment-and-htlc-transaction-test-vectors-anchors-zero-fee-htlc-tx)
```

Reply:
The link still doesn't work in the rendered markdown, I believe it should be:

`#appendix-g-commitment-and-htlc-transaction-test-vectors-anchors_zero_fee_htlc_tx`